samples/bluetooth/mesh/light/src/main.c

```diff
@@ -51,4 +52,8 @@ void main(void)
 	if (err) {
 		printk("Bluetooth init failed (err %d)\n", err);
 	}
+
+	if (IS_ENABLED(CONFIG_BT_MESH_LIGHT_SAMPLE_NRF52_DFU)) {
```

Comment:
Could this use CONFIG_MCUMGR_SMP_BT instead? Or will that add thingy:53 too. Or could it then && !thingy:53

Reply:
Seee above comment ⬆️